frontend/src/components/common/CreateResourceButton.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { getCluster } from '../../lib/cluster';
+import { apply } from '../../lib/k8s/apiProxy';
+import { KubeObjectInterface } from '../../lib/k8s/cluster';
+import { clusterAction } from '../../redux/clusterActionSlice';
+import { EventStatus, HeadlampEventType, useEventCallback } from '../../redux/headlampEventSlice';
+import { ActionButton, EditorDialog } from '../common';
+import { BASE_EMPTY_CONFIG_MAP } from '../configmap/storyHelper';
+import { LEASE_DUMMY_DATA } from '../lease/storyHelper';
+import { BASE_RC } from '../runtimeClass/storyHelper';
+import { BASE_EMPTY_SECRET } from '../secret/storyHelper';
```

Comment:
I guess it might make sense to copy these default structures in here, because their use case is different. I worry someone might change the story in the future not realizing they are not just changing a test... but data that is used to create things.